aces/Loris
# [Datadict] filter issue when table name is different then test_name

## Brief summary of changes
The Data Dictionary allows for filtering on the data from the `parameter_type` table. The `parameter_type` table is built by the `data_dictionary_builder` script which parses the LINST format of any instrument to generate the table entries. the LINST format of a PHP instrument is generated by the `lorisform_parser` script which writes the TABLE of the instrument into the LINST file it creates...

result: The table name is stored in the parameter_type SQL table, and the datadict module expects to filter on test name causing any filtering by instrument to return 0 results when the testname and the table name of an instrument differs.


#### Testing instructions (if applicable)

1. create or modify an instrument to have a testname different then it's table name. (both parameters are usually defined in the instrument class' setup function)
2. run the lorisform_parser on your instrument and the data_dict_builder as instructed in the LORIS wiki
3. navigate to the datadict module and try to filter by the instrument you have chosen

Before these changes, the filter will always return 0 results. with these changes you will see all your fields for that instrument

#### Link(s) to related issue(s)

* Fixes issues for both CBIGR and CCNA
